debian/postinst

```diff
@@ -3,6 +3,8 @@ set -e
 
 [[ "$1" == "configure" ]] || exit 0
 
+sudo npm install -g sqlite3 --unsafe-perm
```

Comment:
We'd want to install it into lounge's folder, not globally. I don't want to rely on global npm install nor leave files all over the place (removing lounge won't remove sqlite3 npm).

Reply:
Yeah totally agree!
Do you know how I can do this? I would say with `sudo npm install --unsafe-perm --prefix /usr/lib/node_modules/thelounge sqlite3`, does that seem correct to you?

Reply:
Probably.

Reply:
Alright, should be fine now. And it even passes CI 🤷‍♂️